bezalel001/Maintenance-Tracker
# Ft admin login #157420296

**What does this PR do?**
- Provides a simple login form for admin

**Description of Task to be completed?**
- A a super user should be able to log in to the admin website

**How should this be manually tested?**
- Navigate to the admin website
- You should see a simple username and password inputs form
**Any background context you want to provide?**

**What are the relevant pivotal tracker stories?**
An admin should be able to login to the application admin site - #157420296
